tests/Microsoft.Bot.Builder.Transcripts.Tests/Microsoft.Bot.Builder.Transcripts.Tests.csproj

```diff
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+

```

Comment:
I believe this file is missing many of the build directives we need. Please check the other csproj files and bring over the relevant bits.

Reply:
We update the `.csproj` files to keep only the relevant code.